crates/bevy_window/src/window.rs

```diff
@@ -126,13 +126,24 @@ pub struct Window {
     /// Note: This does not stop the program from fullscreening/setting
     /// the size programmatically.
     pub resizable: bool,
+    /// Enables and disables window control buttons.
+    ///
+    /// ## Platform-specific
+    ///
+    /// **`iOS`**, **`Android`**, and the **`Web`** do not have window control buttons.
+    ///
+    /// macOS note: When `resizable` is set to `false` the maximize button will be
+    /// disabled regardless of the value of the `EnabledButtons` struct. Additionaly, when
+    /// `resizable` is set to `true` the window will be maximized when its bar is double-clicked
+    /// regardless of whether the maximize button is enabled or not.
+    pub enabled_buttons: EnabledButtons,
     /// Should the window have decorations enabled?
     ///
     /// (Decorations are the minimize, maximize, and close buttons on desktop apps)
     ///
-    //  ## Platform-specific
-    //
-    //  **`iOS`**, **`Android`**, and the **`Web`** do not have decorations.
+    /// ## Platform-specific
+    ///
+    /// **`iOS`**, **`Android`**, and the **`Web`** do not have decorations.
     pub decorations: bool,
```

Comment:
Maybe this should be in the same struct as `EnableButton`, and maybe resizable should be in it too (with a different name for the struct)? I don't have a strong opinion on this just opening the conversation.

Reply:
I think I understand where you are coming from, but without a solid reason for why Bevy would change the abstractions, to me it seems beneficial to keep the API similar to `winit`'s (where resizable, enabled_buttons and decorations are separate things).

Reply:
Fair enough, that's a valid reason to keep it that way!